de.tudarmstadt.ukp.wikipedia.mwdumper/src/main/java/de/tudarmstadt/ukp/wikipedia/mwdumper/importer/XmlDumpReader.java

```diff
@@ -33,6 +33,7 @@
 import java.util.Map;
 import java.util.TimeZone;
 
+import javax.xml.XMLConstants;
```

Comment:
Not sure, but it did not otherwise compile on Eclipse Oxygen with JDK8 (using import maven from scm)